packages/dapper/src/dapper.ts

```diff
@@ -112,6 +122,9 @@ export class Dapper implements DapperInterface {
   getCommunities = NotImplemented;
   getCommunity = NotImplemented;
   getPackage = NotImplemented;
+  getPackageDependencies = NotImplemented;
+  getServiceAccount = NotImplemented;
   getTeam = NotImplemented;
   getUser = NotImplemented;
+  getUserSettings = NotImplemented;
```

Comment:
Is this not missing some of the newly added functions? `getServiceAccountList` and `getTeamList` were added to the interface but not here.